### ISSUE TITLE
Why not increase the training times of epoch  in the  Few-shot Tuning training stage？

### ISSUE THREAD

**Issue by zhouzheng1123** [lixiang-ucas/FSODM#4]
I would like to ask Why not increase the training times of epoch  in the  Few-shot Tuning training stage？Will the increase of training times improve the AP of novel class？Thank you

**Comment by zhouzheng1123** [comment]
and i want to know why the code you are based had the pretrained model in the training time ,but your code did not have, I may not know much about this. Please help me.

**Comment by wxhnibr** [comment]
> and i want to know why the code you are based had the pretrained model in the training time ,but your code did not have, I may not know much about this. Please help me.

Are your results similar to the paper? Can you tell us some training details? For example: batch size, learning rate, number of training rounds
